dm/worker/server_test.go

```diff
@@ -58,7 +58,7 @@ func createMockETCD(dir string, host string) (*embed.Etcd, error) {
 	cfg.LCUrls = []url.URL{*lcurl}
 	cfg.ACUrls = []url.URL{*lcurl}
 	cfg.Logger = "zap"
-	metricsURL, _ := url.Parse(tempurl.Alloc())
+	metricsURL, _ := url.Parse("host://" + tempurl.Alloc()[len("http://"):])
```

Comment:
Why do we using a `host` scheme here?

Reply:
refined in https://github.com/pingcap/dm/pull/453/commits/e6f40e550f45ef72186f6f9853e3e9b31b797b6e